samtranslator/model/api/api_generator.py

```diff
@@ -93,6 +96,11 @@ def _construct_rest_api(self):
             raise InvalidResourceException(self.logical_id,
                                            "Specify either 'DefinitionUri' or 'DefinitionBody' property and not both")
 
+        if self.open_api_version:
+            if re.match(SwaggerEditor.get_openapi_versions_supported_regex(), self.open_api_version) is None:
+                raise InvalidResourceException(
+                    self.logical_id, "The OpenApiVersion value must be of the format 3.0.0")
```

Comment:
nit: `3.0.0` doesn't seem like a format, but I think this is still understandable.